### DISCUSSION TITLE
Is there a plan to address vm2 critical security vulnerability for npm

### DISCUSSION THREAD

**Discussion by poojakharel** [nest-modules/mailer#1037] [Q&A]
I see there was an attempt to fix it for yarn but I still see the vulnerability for npm install, is there a plan to address it soon in future? I want to use it for one of the projects and am kinda blocked by it.

**Comment by sswayney** [comment]
There is a finished PR but we need a release. :/ 

https://github.com/nest-modules/mailer/pull/1021

**Comment by juandav** [comment]
The last added changes have been released.
